### PR TITLE
Split all files into 'app' and 'packages' directories

### PR DESCRIPTION
**Pull request by mrtnzlml**
Closes https://github.com/kiwicom/mobile/issues/642

Our Playground is really pissing me off now.

### CODE REVIEW

File: scripts/activatePlayground.js [hunk 1]
```diff
@@ -28,33 +28,40 @@ YellowBox.ignoreWarnings([
...
+  },
+);
+
+Promise.all([appPromise, packagePromise]).then(() => {
+  const playgroundImportsFileContent = [
+    '// @flow',
+    '',
+    playgroundTestFiles
+      .map(
+        // Replacing app from filepath with ../.. which is the path from playgroundList component to 'packages' folder
```
Comment (tbergq) [inline scripts/activatePlayground.js]: This comment does not seem to match what is happening

**Comment by tbergq** [comment]
> Our Playground is really pissing me off now.

What should we do with it?

**Comment by mrtnzlml** [comment]
Invest more time...

**Comment by tbergq** [comment]
> Invest more time...

😄

File: scripts/activatePlayground.js (hunk 1 above)
Comment (mrtnzlml) [reply scripts/activatePlayground.js]: Fixed. I would appreciate if you could just fix these minor issues and typos if you are trying it locally. You are a merge master - you can do it... 🙂

Comment (tbergq) [reply scripts/activatePlayground.js]: Sure, makes sense, I will do it for the next ones 👍

**Pull request merged**